spec/paperclip/attachment_registry_spec.rb

```diff
@@ -54,21 +54,50 @@
         greeter: { ciao: 'greeting' }
       }
       foo = Class.new
-      Paperclip::AttachmentRegistry.register(foo, :avatar, { yo: 'greeting' })
-      Paperclip::AttachmentRegistry.register(foo, :greeter, { ciao: 'greeting' })
+      Paperclip::AttachmentRegistry.register(foo,
```

Comment:
Please change the formatting of all these `register` lines to match the thoughtbot style guides: https://github.com/thoughtbot/guides/blob/master/style/ruby/sample.rb#L61